client/cmd/dexcctl/config.go

```diff
@@ -110,16 +110,17 @@ func configure() (*config, []string, bool, error) {
 	}
 
 	if cfg.RPCCert == "" {
-		cfg.RPCCert = filepath.Join(appDir, defaultRPCCertFile)
+		// Check in ~/.dexcctl first.
+		cfg.RPCCert = cleanAndExpandPath(filepath.Join(appDir, defaultRPCCertFile))
+		if !fileExists(cfg.RPCCert) { // Then in ~/.dexc
+			cfg.RPCCert = cleanAndExpandPath(filepath.Join(dexcAppDir, defaultRPCCertFile))
+		}
 	}
 
 	if cfg.RPCAddr == "" {
 		cfg.RPCAddr = defaultRPCAddr
 	}
 
-	// Handle environment variable expansion in the RPC certificate path.
-	cfg.RPCCert = cleanAndExpandPath(cfg.RPCCert)
```

Comment:
I just realized this is still needed in case the user *does* specify something and the branch above isn't followed.